### DISCUSSION TITLE
Hey, the icon you use, can you get it somewhere else ?

### DISCUSSION THREAD

**Discussion by Manniala** [BlafKing/sd-civitai-browser-plus#67] [Q&A]
Hey, thx for the Extension, but got a question, any chance you can hoste the icon someplace else :)?

My MalwareByte is NOT happy about the host you use for the .svg, its not a big deal if not, but...

If you look at the url below, it shows that most sites like, W3 Tools, Norton, Google, McAfee... all say the site is safe, but now a day you never know, so if i get a warning, i rather not go there :)
The reason why it is popping up with that info, i have no clue, i have tried to look into it, but cant find a reason why MalwareByte is not trusting the site, but may not be the only one that have this issue, since all you get is a icon not showing, what was the reason i looked into this :D

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/32381860/90670537-c041-46f7-9e6e-cb207f185f04)

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/32381860/32f9c1ac-4f4b-4086-86eb-de978e24d748)

Source: https://webrate.org/site/svgur.com/

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/32381860/5d70a4f8-2802-4ad0-8832-929d9b58d7cd)

Did a refresh:

![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/32381860/c472572b-9aec-453b-8a6b-dba6d2205648)

**Comment by BlafKing** [comment]
I've now switched to using github gists for hosting the svg icons in the latest update, that should fix your issue! :)

**Reply by Manniala** [reply]
Thx :)
